nuggetsapp/models.py

```diff
@@ -58,6 +64,16 @@ def add_new_nugget_for_user(cls, nugget, user, is_owner=False):
                 is_deleted=False)
 
     @classmethod
-    def get_nuggets_by_user(cls, user):
-        return [x.nugget for x in cls.objects.filter(user = user)]
-
+    def get_nugget_users_by_user(cls, user, exclude_deleted =True):
+        q_objects = Q(user = user)
+        q_objects.add(Q(is_deleted = False), Q.AND) if exclude_deleted
+        return cls.objects.filter(q_objects)
+                
+    @classmethod
+    def get_todays_review_nugget_users_by_user(cls, user, exclude_deleted =True):
+        review_interval_days = [1, 3, 7, 14, 30, 90, 180, 360, 720]
+        review_dates = [date_for_x_days_before_today(n) for n in review_interval_days]
+        q_objects = Q(user = user) & Q(created_at__date__in = review_dates)
```

Comment:
Did you say this rounds to YYYY/MM/DD or strips out hh:mm:ss?  Would be nice to explicitly comment what this does since not obvious.